src/argilla/server/apis/v0/handlers/text2text.py

```diff
@@ -73,32 +73,17 @@ async def bulk_records(
     ) -> BulkResponse:
         task = task_type
         workspace = common_params.workspace
-        try:
-            dataset = await datasets.find_by_name(
-                current_user,
-                name=name,
-                task=task,
-                workspace=workspace,
-            )
-            await datasets.update(
-                user=current_user,
-                dataset=dataset,
-                tags=bulk.tags,
-                metadata=bulk.metadata,
-            )
-        except EntityNotFoundError:
-            dataset = CreateDatasetRequest(name=name, workspace=workspace, task=task, **bulk.dict())
-            dataset = await datasets.create_dataset(user=current_user, dataset=dataset)
+
+        dataset = await datasets.find_by_name(current_user, name=name, task=task, workspace=workspace)
+
+        await datasets.update(user=current_user, dataset=dataset, tags=bulk.tags, metadata=bulk.metadata)
```

Comment:
maybe we should move this also out of the bulk endpoints and do it just one time in `rg.log` before adding the records

Reply:
We can tackle this in a separate PR.